server/channels/app/platform/web_conn.go

```diff
@@ -865,6 +882,11 @@ func (wc *WebConn) ShouldSendEvent(msg *model.WebSocketEvent) bool {
 	return true
 }
 
+func (wc *WebConn) notInChannelAndThread(val string) bool {
+	return (wc.isSet(wc.GetActiveChannelID()) && val != wc.GetActiveChannelID()) &&
+		(wc.isSet(wc.GetActiveThreadChannelID()) && val != wc.GetActiveThreadChannelID())
+}
```

Comment:
Would it make sense to rename val to `channelID`? Also, do we support this channel id to be empty? Asking since I see you are calling `isSet` on that as well.

Reply:
> Would it make sense to rename val to channelID?

sure

> Also, do we support this channel id to be empty? Asking since I see you are calling isSet on that as well.

No we don't. It's there for consistency but I can remove it if you prefer.

Reply:
Okay, consistency makes sense, not a problem of course.